ibm_mq/assets/configuration/spec.yaml

```
@@ -166,6 +166,12 @@ files:
         value:
           example: False
           type: boolean
+      - name: try_normal_ssl
```

A little confused about the name. Is this option to try a normal SSL connection? Or a normal connection before SSL?
```suggestion
      - name: try_normal_connection
```